client/src/views/users/UserListDisplay.vue

```diff
@@ -69,9 +69,14 @@ const hotkeyManager: HotkeyManager = inject(HotkeyManagerKey)!;
 let hotkeys: Hotkeys | null = null;
 
 const selectedCount = ref(0);
+const activeUsersCount = props.users.getUsers().filter((user) => !user.isRevoked()).length;
```

Comment:
Conflicts with master were not properly resolved when rebasing, `selectedCount` is not present on master, and I think updates to the code of this component is useless.

Reply:
Nevermind.